ksql-common/src/main/java/io/confluent/ksql/schema/ksql/LogicalSchema.java

```diff
@@ -148,6 +156,13 @@ public ConnectSchema valueConnectSchema() {
    * @param columnName the column name, where any alias is ignored.
    * @return the value column if found, else {@code Optional.empty()}.
    */
+  public Optional<Column> findValueColumn(final ColumnName columnName) {
+    return doFindColumn(columnName.name(), value);
+  }
+
+  /**
+   * @see #findValueColumn(ColumnName)
+   */
```

Comment:
`@VisibleForTesting`?  Others below too?

Reply:
Looks like I did miss some usages here! I'll clean this up the next PR